drivers/tests/TESTS/mbed_drivers/buffered_serial/CMakeLists.txt

```diff
@@ -4,10 +4,15 @@
 cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
 
 set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../.. CACHE INTERNAL "")
-set(TEST_TARGET mbed-drivers-buffered-serial)
+set(APP_TARGET mbed-drivers-buffered-serial)
 
 include(${MBED_PATH}/tools/cmake/mbed_greentea.cmake)
 
-project(${TEST_TARGET})
+project(${APP_TARGET})
 
-mbed_greentea_add_test(TEST_NAME ${TEST_TARGET})
+if (DEFINED MBED_TEST_LINK_LIBRARIES)
+    separate_arguments(MBED_TEST_LINK_LIBRARIES)
```

Comment:
can't we just invoke a macro, and macro does the magic (this ifdefined should be in the macro not in the app) ? argument link libs would be optional, possible?

Reply:
Until we set as cache string, it will not be defined to access